src/components/Trailers/GridCard/VideoCard/VideoCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Card, CardContent, CardMedia, Typography, Box, Modal } from "@mui/material";
+import PlayCircleOutlineIcon from "@mui/icons-material/PlayCircleOutline";
+import { useDispatch } from "react-redux";
+import { Endpoints } from "../../api/constants/endpoints";
+
+interface Props {
+  filmName: string
+  img: string
+  trailerKey: string
+};
+
+export const VideoCard = ({ filmName, img, trailerKey }: Props) => {
+
+  const style = {
+    position: "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 1467,
+    height: 824,
+    bgcolor: "black",
+    ":focus-visible": {
+      outline: "none",
+    }
+  };
+
+  const [open, setOpen] = React.useState(false);
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
```

Comment:
you can simplify logic rewriting with toggle

Reply:
done